crates/ark/src/lsp/completions/completion_item.rs

```diff
@@ -416,14 +425,14 @@ pub(super) unsafe fn completion_item_from_lazydata(
     let promise_strategy = PromiseStrategy::Simple;
 
     // Lazydata objects are never functions, so this doesn't really matter
-    let parameter_hints = ParameterHints::Enabled;
+    let parameter_hints = ParameterHints::Disabled;
```

Comment:
This is unrelated and "doesn't really matter", but if we're going to stick an arbitrary value in here, `Disabled` just makes more sense to me 🤷‍♀️